dev/com.ibm.ws.kernel.boot.cmdline/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -670,6 +670,14 @@ error.zosProcStart.jobname.invalid=CWWKE0943E: The z/OS STC procedure {0} failed
 error.zosProcStart.jobname.invalid.explanation=The STC job name specified by the WLP_ZOS_JOBNAME property in the server.env file is not valid. The job name must be no longer than 8 characters and must not contain a comma.
 error.zosProcStart.jobname.invalid.useraction=Verify that the job name is no longer than eight characters and that it does not contain a comma.
 
+error.server.pause.command.port.disabled=CWWKE0944E: A pause request was received for server {0}, but the server command port is disabled. The request cannot be processed. 
+error.server.pause.command.port.disabled.explanation=The server command port is required for communication with a running server.  The command port is disabled for this server, therefore the pause request could not be processed.  Note that the command port is disabled by default when a server is started as a started task on z/OS.
+error.server.pause.command.port.disabled.useraction=If the server is running as a started task on z/OS, this dump can be requested from the operator console.  See the information center for details.
```

Comment:
@awisniew90 I saw that there were several existing errors that this is based on, but still had some edits and questions (below).

error.server.pause.command.port.disabled=CWWKE0944E: A pause request was received for server {0}, but the server command port is disabled, so the request cannot be processed. **[Add brief user action]**
error.server.pause.command.port.disabled.explanation=The server command port is required for communication with a running server.  The command port is disabled for this server, so the pause request could not be processed.  Note that the command port is disabled by default when a server is started as a started task on z/OS.
error.server.pause.command.port.disabled.useraction=If the server is running as a started task on z/OS, you can request this dump **(what dump? and why?)** from the operator console. For more information, see the documentation.
 - **This user action appears in the documentation. Which documentation should they be checking, the z/OS documentation? Note that "information center" is an outdated term.**
- **What if they aren't on z/OS? What action should they take? Presumably something like "Enable the server command port."?**

Reply:
@cjcarpen - "dump" should be "pause" and "resume" (my mistake). I also added a brief user action to the message and a better description in the useraction section. The useraction section now recommends enabling the command port and/or running the command from the operator console if on z/OS. In this case, the documentation is both z/OS and non z/OS.  Would saying "see the Knowledge Center for more information" be sufficient?